orc8r/protos/magmad.proto

```diff
@@ -93,6 +93,25 @@ message LogLine {
   string line = 1;
 }
 
+message StatelessAgwMode {
+  enum AgwMode {
```

Comment:
protoc will scope embedded enum type definitions and prefix it with message type name, so - AgwMode will look redundant: StatelessAgwMode_AgwMode, you can probably do StatelessAgwMode:Type and Value, it'll look nicer:

message StatelessAgwMode {
    enum Type {
    ...
    type Type;
    ...